app/User.php

```diff
@@ -12,7 +12,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'name', 'email', 'password', 'type',
```

Comment:
I dont think the type should be a fillable in the user, for security reasons.
in my opinion, a type can only be set by an admin through his panel and all users are normal at first when they are registered to the system.
The first admin of the system should be supplied with the system when it is delivered but not created